### DISCUSSION TITLE
Branching Stragegy (Contributing)

### DISCUSSION THREAD

**Discussion by justinstroud-campmeeker** [pmd/pmd#4879] [Q&A] [answered]
Hey there. I am working on a Salesforce Metadata module and I'm curious about the branching strategy. I see that it says to create pull requests off of master. This, meaning, I should not create a branch or anything. Just a PR off of master and let the reviews flow. Is this correct?

**Comment by jsotuyod** [comment, marked as answer]
Ideally, fork the repository, branch off from master, submit the PR back to master.
